_tutorials/preferences-api.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: develop
+title: Using the Preferences API
+
+summary: The Fyne toolkit includes a Preferences API that makes it easy for developers to store values locally on the computer and then grab them again when necessary.
+
+order: 50
+redirect_from: /develop/preferences-api.html
+---
+
+## Using the Preferences API
+
+---
+
+Storing user configurations and values is a common task for application developers, but implementing it across multiple platforms can be tedious and time-consuming. To make it easier, Fyne has an API for storing values on the filesystem in a clean and understandable way while the complex parts are handled for you.
+
+Lets start with the setup of the API. It is part of the [Preferences](https://pkg.go.dev/fyne.io/fyne?tab=doc#Preferences) interface where storage and loading functions exist for values of Bool, Float, Int and String. They each consist of three different functions, one for loading, one loading with a fallback value and lastly, one for storing values. An example of the three functions and their behaviour can be seen below for the String type:
+```go
+// String looks up a string value for the key
+String(key string) string
+// StringWithFallback looks up a string value and returns the given fallback if not found
+StringWithFallback(key, fallback string) string
+// SetString saves a string value for the given key
+SetString(key string, value string)
+```
+
+These functions can be accessed through the created application variable and the `Preferences()` interface on top of it. Please note that it is necessary to create the apps with a unique ID (usually like a reversed url). This means that the application will need to be created using `app.NewWithID()` to have its own place to store values. It can roughly be used like the example below:
```

Comment:
Oh, sorry - but `Preferences()` is a method not an interface.

```suggestion
These functions can be accessed through the created application variable and calling the `Preferences()` method on. Please note that it is necessary to create the apps with a unique ID (usually like a reversed url). This means that the application will need to be created using `app.NewWithID()` to have its own place to store values. It can roughly be used like the example below:
```